client/src/components/common/ProfileCard/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import avatarImg from '../../../assets/img_avatar.png';
+import Button from '../Button';
+import './style.css';
+
+class ProfileCard extends Component {
+  state = {};
+
+  render() {
+    const {
+      name, location, language, dialect,
+    } = this.props;
+    return (
+      <div className="profileCard__main">
+        <img src={avatarImg} alt="avatarImg" className="avatar__img" />
+        <h1 className="profileCard__name">{ name }</h1>
+        <h1 className="profileCard__location">{ location }</h1>
+        <h1 className="profileCard__lang">
+          { language }
+        -
+          { dialect }
+        </h1>
+        <Button className="button__profileCard" value="Show Profile" onClick={null} />
```

Comment:
we should add the onclick function in this PR, and since we know the endpoint for the profile, we can tell it to redirect to that with react-router.